db/seeds.rb

```diff
@@ -6,8 +6,12 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-# Load Nationalities
-
+# Load Nationalities. # TODO, move nationalities out of yaml.
```

Comment:
Todo?

Reply:
A reminder for myself to move the nationalities into an initializer like the other stuff. I've created a ticket for it so i'll remove this comment